### PR TITLE
Fix: Fix View Preference Modal not displaying in View Profile

### PR DESCRIPTION
**Pull request by danthecm**
# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What were you told to do?

Fix Modals

## What did you do?

Fixed View Profile Modal

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

## Before

https://user-images.githubusercontent.com/52335952/207524676-55f35851-3256-42fd-b9e6-1720966721df.mp4

## After

https://user-images.githubusercontent.com/52335952/207524777-3fd47994-d36d-41f2-8bbd-03d433e3831b.mp4

### CODE REVIEW

**Comment by danthecm** [comment]
@billmal071

File: packages/main/src/components/protected/topbar/components/ProfileMore.jsx [hunk 1]
```diff
@@ -64,7 +64,8 @@ export const Dropdown = ({ setDropdown }) => {
         <div className={styles.topSection}>
           <p
             onClick={() => {
-              setReusableModal("preference");
+              // setReusableModal("preference");
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/ProfileMore.jsx:67]: Remove this commented out code

File: packages/main/src/components/protected/topbar/components/ProfileMore.jsx [hunk 2]
```diff
@@ -11,11 +11,11 @@ import useOnClickOutside from "../hooks/useClickOutside";
 
 import { useTranslation } from "react-i18next";
 
-export const Dropdown = ({ setDropdown }) => {
+export const Dropdown = ({ setDropdown, setModal }) => {
   const { t } = useTranslation();
 
   const { toggleModalState } = useContext(ProfileContext);
-  const [modal, setModal] = useState("");
+  // const [modal, setModal] = useState("");
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/ProfileMore.jsx:18]: Remove this commented out code

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/protected/topbar/components/ProfileMore.jsx (hunk 2 above)
Comment (danthecm) [reply packages/main/src/components/protected/topbar/components/ProfileMore.jsx]: Commented Codes have been remove @JulianaSau

File: packages/main/src/components/protected/topbar/components/ProfileMore.jsx (hunk 1 above)
Comment (danthecm) [reply packages/main/src/components/protected/topbar/components/ProfileMore.jsx]: Commented code have been removed

Comment (danthecm) [reply packages/main/src/components/protected/topbar/components/ProfileMore.jsx]: Please review again @JulianaSau

**Pull request merged**